packages/components/pages/f-checkout/src/components/Checkout.vue

```diff
@@ -449,11 +449,14 @@ export default {
         },
 
         /**
-         * Update address lines if localStorage is populated.
+         * Update address lines if it is saved in storage (cookie or local storage), varies among tenants.
          *
          * */
-        loadAddressFromLocalStorage () {
-            const address = addressService.getAddressFromLocalStorage(this.tenant);
+        loadAddressFromStorage () {
+            const shouldLoadAddressFromLocalStorage = addressService.shouldLoadAddressFromLocalStorage(this.tenant);
```

Comment:
IMHO the Vue component doesn't need to know how the address is retrieved - the service needs to decide where to get it from (localstorage or cookie). Otherwise, you have the getting address logic split and any change in the service may require changing the component too.

It's cleaner to do 
```
loadAddressFromStorage () {
     const address = addressService.getAddress(this.tenant) 
...
```
In reality, the component needs an address and it's the service's responsibility to provide one, hiding the implementation details.